tests/utils.go

```diff
@@ -2719,6 +2719,13 @@ func SkipIfNotUseNetworkPolicy(virtClient kubecli.KubevirtClient) {
 	}
 }
 
+func SkipIfNoGpuDevicePlugin(virtClient kubecli.KubevirtClient) {
```

Comment:
AFAIK the current approach for functional tests that require additional resources that may be unavailable is:

1) don't use skips;
2) instead, have a separate suite just for the affected test cases, and configure CI jobs to not trigger the suite in the gate;
3) ideally, add a CI job that would run on a machine that has the corresponding resources and trigger the separate test suite there only.

So I think at the very least this skip should be removed. Since you have a suite for GPU test cases, it's fine as long as you make sure gate jobs don't trigger it.

Reply:
Removed skips. Configure CI jobs to not trigger GPU test suite.